pkg/client/client.go

```diff
@@ -687,15 +687,16 @@ func FormatStatusResponse(w io.Writer, sr *models.StatusResponse, sd StatusDetai
 		if sr.Encryption.Msg != "" {
 			fields = append(fields, sr.Encryption.Msg)
 		} else if wg := sr.Encryption.Wireguard; wg != nil {
+			fields = append(fields, fmt.Sprintf("[NodeEncryption: %s", wg.NodeEncryption))
 			ifaces := make([]string, 0, len(wg.Interfaces))
 			for _, i := range wg.Interfaces {
 				iface := fmt.Sprintf("%s (Pubkey: %s, Port: %d, Peers: %d)",
 					i.Name, i.PublicKey, i.ListenPort, i.PeerCount)
 				ifaces = append(ifaces, iface)
 			}
-			fields = append(fields, fmt.Sprintf("[%s]", strings.Join(ifaces, ", ")))
+			fields = append(fields, fmt.Sprintf("%s]", strings.Join(ifaces, ", ")))
 		}
 
-		fmt.Fprintf(w, "Encryption:\t%s\n", strings.Join(fields, "\t"))
+		fmt.Fprintf(w, "Encryption:\t%s\n", strings.Join(fields, ", "))
```

Comment:
I think we should keep the tab between the state (i.e. Encryption.Mode) and the details. We're not super consistent in the output, but that's what other subsystems (e.g. Hubble, Kubernetes) also seem to use a tab instead of a comma.

Therefore, I'd propose something like this:
```go
fmt.Fprintf(w, "Encryption:\t%s\t%s\n", sr.Encryption.Mode, strings.Join(fields, ", "))
```

(and of course remove `sr.Encryption.Mode` from the `fields` array on line 685)

Reply:
> I think we should keep the tab between the state (i.e. Encryption.Mode) and the details. We're not super consistent in the output, but other subsystems (e.g. Hubble, Kubernetes) also seem to use a tab instead of a comma.
> 
> Therefore, I'd propose something like this:
> 
> ```go
> fmt.Fprintf(w, "Encryption:\t%s\t%s\n", sr.Encryption.Mode, strings.Join(fields, ", "))
> ```
> 
> (and of course remove `sr.Encryption.Mode` from the `fields` array on line 685)

Ok. I fixed it
Thanks